### PR TITLE
Updated join.md form

### PR DESCRIPTION
**Pull request by sakethkoduru**
Made changes to join.md form

### CODE REVIEW

File: docs/about/join.md [hunk 1]
```diff
@@ -41,24 +45,33 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
...
   <label for="email_address">UCSF email address</label> <span style="color: red">*</span><br>
-  <input type="email" id="email" name="email" autocomplete="email" placeholder="Enter email" size="30" required><br>
+  <input type="email" id="email" name="email" autocomplete="email" placeholder="Enter email" size="30" required><br> 
+  <small>(**External collaborators only: Please provide your preferred email address)</small>
  </div>
  
- <div class="form-group">
-  <label>UCSF affiliation</label> <span style="color: red">*</span><br>
+ <div class="form-group"> 
+  <label for="affiliation">UCSF affiliation</label> <span style="color: red">*</span><br>
```
Comment (HenrikBengtsson) [inline docs/about/join.md:53]: I think it's great that you've tried to add the `for` attribute to the HTML label tag.  I wasn't aware of it.  However, from <https://www.w3schools.com/tags/tag_label.asp>, it looks like it is meant to specify an `id` attribute but It looks like what you've specified, `"affiliation"`, matches the `name` attribute.  Are you sure what you've added, works?

File: docs/about/join.md [hunk 2]
```diff
@@ -41,24 +45,33 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
...
   </fieldset>
  </div>
-
+ <div class="form-group">
+  <label for="other_desc">If 'Other', please clarify </label><br>
+  <input type="text" id="other_desc" name="other_desc" placeholder="Collaborating with, ..." size="30" minlength="2"><br>
+ </div>
+ 
+ <div class="form-group">
+  <label for="student_affiliation"> If 'Student', please select appropriate option</label> <br>
```
Comment (HenrikBengtsson) [inline docs/about/join.md:69]: Same here, there's no HTML tag with an `id = "student_affiliation"`.  Is this correct?

File: docs/about/join.md [hunk 3]
```diff
@@ -71,17 +84,20 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
 
  <div class="form-group">  
   <label for="ucsf_id">UCSF Employee/Student ID</label> <span style="color: red">*</span><br>
-  <input type="text" id="ucsf_id" name="ucsf_id" placeholder="02xxxxxxx or 'none'" pattern="(\d{9}|none)" maxlength="9" size="15" required> <small>(nine digits; to find yours, go to <a href="https://myaccess.ucsf.edu/myid">https://myaccess.ucsf.edu/myid</a>)</small><br>
+  <input type="text" id="ucsf_id" name="ucsf_id" placeholder="02xxxxxxx or 'none'" pattern="(\d{9}|none)" maxlength="9" size="15" required><br> 
+  <small>(**Nine digits: To find yours, go to <a href="https://myaccess.ucsf.edu/myid">https://myaccess.ucsf.edu/myid</a>)</small><br>
```
Comment (HenrikBengtsson) [inline docs/about/join.md:88]: What's the purpose of `**`?  Is that meant to be Markdown code for `<strong>` or maybe `<em>`?

File: docs/about/join.md [hunk 4]
```diff
@@ -92,13 +108,12 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
     <input type="checkbox" id="ucsf_compute_environments_2" name="ucsf_compute_environments" value="qb3"/> QB3 (now deprecated)<br>
     <input type="checkbox" id="ucsf_compute_environments_3" name="ucsf_compute_environments" value="radiology"/> Radiology<br>
     <input type="checkbox" id="ucsf_compute_environments_4" name="ucsf_compute_environments" value="sali"/> Sali Lab<br>
-    <input type="checkbox" id="ucsf_compute_environments_5" name="ucsf_compute_environments" value="tipcc"/> TIPCC (Cancer Center compute cluster)<br>
```
Comment (HenrikBengtsson) [inline docs/about/join.md:95]: Why was TIPCC dropped?  Mistake?

Comment (sakethkoduru) [reply docs/about/join.md:95]: @HenrikBengtsson Since TIPCC didn't join Wynton, the UID and GID provided will be not relevant. It was intentional.

Comment (HenrikBengtsson) [reply docs/about/join.md:95]: Moving this over to Gitea.

File: docs/about/join.md (hunk 1 above)
Comment (sakethkoduru) [reply docs/about/join.md:53]: It worked, but I will change it to be consistent with the rest of the label tags

Comment (HenrikBengtsson) [reply docs/about/join.md:53]: What do you mean by "It worked"?

It's not clear to me how to map one label to a set of radio elements, but then I also know very little about this.  If you can figure it out, it'll probably improve the form. (e.g. accessibility)

Comment (sakethkoduru) [reply docs/about/join.md]: Updated code and validated

File: docs/about/join.md (hunk 2 above)
Comment (sakethkoduru) [reply docs/about/join.md]: Updated code and validated

File: docs/about/join.md (hunk 3 above)
Comment (sakethkoduru) [reply docs/about/join.md]: made changes as per suggestion

**Comment by HenrikBengtsson** [comment]
Thxs :+1:

**Pull request merged**